src/plugins/mer/mersettings.cpp

```diff
@@ -230,7 +238,34 @@ void MerSettings::save()
         settings->setValue(QLatin1String(QML_LIVE_BENCH_LOCATION_KEY), m_qmlLiveBenchLocation);
     settings->setValue(QLatin1String(ASK_BEFORE_STARTING_VM), m_askBeforeStartingVmEnabled);
     settings->setValue(QLatin1String(ASK_BEFORE_STARTING_VM), m_askBeforeClosingVmEnabled);
+    settings->endGroup();
+
+    settings->beginGroup(QLatin1String(SETTINGS_PROPERTY_CATEGORY));
+    for (const QByteArray &name : dynamicPropertyNames())
+        settings->setValue(QString::fromLatin1(name), property(name));
+    settings->endGroup();
```

Comment:
I checked all upstream plugins and all seem to provide proper C++ access functions for all properties they implement persistency for. I don't think we should do different here, so I would prefer not doing this and instead provide normal C++ access functions for all properties.

Reply:
Removed this code